### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by DaudKhadijah01**
ready

### CODE REVIEW

**Comment by DaudKhadijah01** [comment]
ready

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,56 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '3.0.1'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails', branch: 'main'
+gem 'rails', '~> 6.1.4'
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3', '~> 1.4'
```
Comment (ngocvt-0484) [inline Gemfile]: please use gem "mysql2", delete sqlite3

File: config/database.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
+#
+default: &default
+  adapter: sqlite3
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  timeout: 5000
+
+development:
```
Comment (ngocvt-0484) [inline config/database.yml]: this file will not be up to GIt, please

```git rm --cached config/database.yml```
and in .gitignore add
```
config/database.yml
```

File: Gemfile (hunk 1 above)
Comment (DaudKhadijah01) [reply Gemfile]: done

Comment (DaudKhadijah01) [reply Gemfile]: image
Screenshot from 2021-08-18 08-51-10
but i cannot open my rails after change the gem

**Comment by DaudKhadijah01** [comment]
ready

**Comment by ngocvt-0484** [comment]
you havent not fix comment yet

**Comment by ngocvt-0484** [comment]
commented

File: Gemfile (hunk 1 above)
Comment (DaudKhadijah01) [reply Gemfile]: image
done

**Comment by DaudKhadijah01** [comment]
> you havent not fix comment yet. 
i am sorry but i do think that i already fix the comments. hence, which comment that i did not fix yet?

**Comment by DaudKhadijah01** [comment]
ready

**Review by huongnt-2545 — COMMENTED** [review]
You haven't apply the comment `Using MySQL` instead of `SQLite3`
If you fixed it, please follow below steps to update the changes on github:
On this branch
- git add .
- git commit --amend --no-edit
- git push origin init_project -f

**Comment by huongnt-2545** [comment]
You can see at tab **Files changed** in pull request
image

**Comment by DaudKhadijah01** [comment]
ready

**Comment by DaudKhadijah01** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged
You can merge this pull request

**Comment by DaudKhadijah01** [comment]
i can not do the pull request for my chapter_3. it stated that my main and chapter_3 are identical. How to resolve it? 

image

image

image

**Comment by DaudKhadijah01** [comment]
ready

**Pull request merged**